extractor/src/test/java/org/schabi/newpipe/extractor/NewPipeTest.java

```diff
@@ -27,6 +29,21 @@ public void testAllServicesHaveDifferentId() throws Exception {
         }
     }
 
+    @Test
+    public void testGetCountryCodeAndGetLanguageCode() {
+        try {
+            HashSet<Integer> servicesId = new HashSet<>();
+            for (StreamingService streamingService : NewPipe.getServices()) {
```

Comment:
I'm not sure why all services are tested.
The implementation is always identical:
https://github.com/lacinoire/NewPipeExtractor/blob/23fa31a1ec572cfb45f0f2fe88cc3844b151aa73/extractor/src/main/java/org/schabi/newpipe/extractor/StreamingService.java#L373-L402